app/src/test/admin/components/AdminAppTests.tsx

```diff
@@ -5,11 +5,11 @@ import {AdminApp, AdminAppComponent} from "../../../main/admin/components/AdminA
 import {AdminRouter} from "../../../main/admin/components/AdminRouter";
 
 describe("AdminApp", () => {
-    it("passes through loggedIn to router", () => {
-        const rendered = shallow(<AdminAppComponent loggedIn={ true } />);
+    it("passes through history to router", () => {
+        const rendered = shallow(<AdminAppComponent loggedIn={ true } history={ "TEST HISTORY" as any } />);
```

Comment:
Maybe remove `loggedIn` here? I know it's now `undefined` because it's derived from state but it's slightly confusing to be testing a legacy prop.